end_to_end/gemma/7b/1_test_gemma.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# This file converts the Gemma checkpoint from kaggle
```

Comment:
I think we should mention that this can be run on any CPU machine with at least XX RAM

Also, I am thinking we mention that `This file is both an integration test that runs once a day on a YY machine`